threedi_modelchecker/checks/other.py

```diff
@@ -324,7 +324,7 @@ def get_invalid(self, session: Session) -> List[NamedTuple]:
             f"""SELECT *
                FROM connection_node AS cn1, connection_node AS cn2
                WHERE
-                   distance(cn1.geom, cn2.geom, 1) < :min_distance
```

Comment:
The third argument, use_ellipsoid, is only suported for long/lat coordinates and will return None's for projected. Because from now on only projected should be allowed, this argument is removed.